src/components/common/CheckWallet/index.tsx

```diff
@@ -30,6 +31,7 @@ const CheckWallet = ({
   noTooltip,
   checkNetwork = false,
   allowUndeployedSafe = false,
+  allowProposer = true,
```

Comment:
Just an observation: All other `allowX` props default to `false` when not set.

I understand that it makes more sense but we need to be aware of this :D